### DISCUSSION TITLE
BYG Causing mod incompatibilities.

### DISCUSSION THREAD

**Discussion by GRKTheGreat** [CorgiTaco/BYG#1211] [Q&A]
I want to disable these from the mod "[19:42:28] [Render thread/ERROR]: [net.minecraft.class_156#method_29191] No data fixer registered for byg:boat359 | [19:42:28] [Render thread/ERROR]: [net.minecraft.class_156#method_29191] No data fixer registered for byg:chest_boat360 | [19:42:28] [Render thread/ERROR]: [net.minecraft.class_156#method_29191] No data fixer registered for byg:man_o_war361 | [19:42:28] [Render thread/ERROR]: [net.minecraft.class_156#method_29191] No data fixer registered for byg:pumpkin_warden" so that the mod hopefully works in my custom modpack. The mod file is encrypted so i can't do what i need to rn.

**Comment by CorgiTaco** [comment]
This is not an issue. Byg is fine

**Comment by shanoaice** [comment]
Data fixer errors can usually be safely ignored.
